### PR TITLE
Update README.adoc for cloud hosted guide instructions

### PR DESCRIPTION
**Pull request by gkwan-ibm**
to address #33

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -111,13 +160,32 @@ After you see the following message in both command-line sessions, both of your
...
+```
+curl http://localhost:9081/inventory/systems/localhost
+```
+{: codeblock}
+
+When you visit this endpoint, you make two GET HTTP requests, one to the **system** service and one to the **inventory**
+service. Both of these requests are configured to be traced, so a new trace is recorded in Jaeger.
+
+To view the traces, go to the **`https://accountname-16686.theiadocker-4.proxy.cognitiveclass.ai`** URL.
+You can view the traces for the inventory or system services under the **Search** tab in the upper left of the page.
```
Comment (dmuelle) [inline README.adoc]: "under the **Search** tab in the upper left of the page."
For accessibility compliance, avoid directional language- "upper left of the page"
If there is not another clear way to give an indication of where to look for the search tab, i think the following is ok:
"You can view the traces for the inventory or system services under the **Search** tab."

File: README.adoc [hunk 2]
```diff
@@ -111,13 +160,32 @@ After you see the following message in both command-line sessions, both of your
...
+Select the services in the **Select a service** menu and click the **Find Traces** button at the end of the section.
```
Comment (dmuelle) [inline README.adoc:187]: just confirming both these are consistent with what the user sees:

**Select a service**
 **Find Traces**
 The first uses sentence capitalization and the second uses Headline Captialization, which is not expected in the same UI, But if that's consistent with what the user sees, no edit is needed

**Review by dmuelle — COMMENTED** [review]
looks good, just a couple minor things to check

File: README.adoc (hunk 1 above)
Comment (gkwan-ibm) [reply README.adoc]: updated

File: README.adoc (hunk 2 above)
Comment (gkwan-ibm) [reply README.adoc:187]: Do use mean "Find Traces" -> "Find traces"? This button is not from us but from the jaegar server. Yes, it is what the user sees.

Comment (dmuelle) [reply README.adoc:187]: ok as long as the text and captialization here are the same as in jaeger, no edit is needed

**Review by dmuelle — APPROVED** [review]
LGTM

**Pull request merged**